dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -639,18 +653,54 @@ public Boolean checkCWL(File content) {
         //go through each line of the file content and find the word patterns as described above
         try {
             List<String> fileContent = java.nio.file.Files.readAllLines(p, StandardCharsets.UTF_8);
-            for (String line : fileContent) {
+            for (int i = 0; i < fileContent.size(); i++) {
```

Comment:
Change this back to an enhanced for loop. 
Depending on what the new section does, I would feed a copy of a portion of the list into your new section (which should be a new method due to its length)